src/components/PuzzleGrid/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react";
+import { Col, Row } from "antd";
+import puzzledImage from "./../../images/habitat.jpg";
+import shuffle from "./../memorygame/shuffle";
```

Comment:
You are right, but I kept like this for now since Sule is also using this function and it might cause a problem when our PRs are merged, I thought it's better to move it after all of our PRs are merged

Reply:
+1 someone please file an issue for the cleanup and make sure its done by end of sprint

Reply:
+1 to majd i mean, i would just use memorygame for now and move it after. or move it in a separate PR right now ( a really small one) and force everyone to merge

Reply:
I created issue for it